structured_experiments/2022.04.28.notebook_template/experiment.MD

```diff
@@ -21,11 +21,22 @@ TODO: Create Template
 ## 1. First Acquire the training data
 
 ### 
-Run `git lfs pull --exclude .git -I  data/storysquad-ground-truth/`
+Run
+- `git lfs pull --exclude .git -I  data/storysquad-ground-truth/`
+
+- `git lfs pull --exclude .git -I  data/kaggle-ground-truth/`
 
 This will use git-lfs to pull down all the client given training images
 
-## 2. Execute the notebook container
+## 2. Execute the Template Experiment

```

Comment:
The Template experiment needs to be run first to have all the correct files in the right places before it is possible to run the notebook template without running into bugs

Reply:
This is a great explanation. Thank you